build.csx

```diff
@@ -143,3 +133,18 @@ public static void RunDotNet(string workingDirectory, string command, string arg
         }
     }
 }
+
+public string GetFakeItEasyVersion()
+{
+    Regex versionMatcher = new Regex(@"""FakeItEasy""\s*:\s*""([^""]+)""", RegexOptions.Compiled);
+    foreach ( string line in File.ReadLines(mainProjectPath) )
+    {
+        var match = versionMatcher.Match(line);
+        if (match.Success)
+        {
+            return match.Groups[1].Value;
+        }
+    }
+
+    throw new InvalidOperationException($"Can't find FakeItEasy version in {mainProjectPath}");
+}

```

Comment:
Wow, I didn't even know it was possible to parse JSON as XML. I don't know if I find it fascinating or horrifying... maybe a bit of both :wink:
I was thinking about using JSON.NET, but maybe it's overkill. If it works this way, it's fine by me. Although I would probably be more specific for finding the dependency (e.g. `root.Element("dependencies").Element("FakeItEasy").Value`)